libs/frontend/feature-build/src/lib/dnd/row-layout.ts

```diff
@@ -38,74 +38,90 @@ export const isRowLayout = (
   return direction === 'row' || direction === undefined
 }
 
-export const handleRowLayoutHover = ({
-  mousePosition,
-  targetMiddle,
-  haveSameParent,
-  dragComponentIndex,
-  targetIndex,
-  dropProps,
-}: IHandleHoverProps) => {
+export const handleRowLayoutHover = (props: IHandleHoverProps) => {
```

Comment:
Hi @sampullman 

I adjusted the handler of hover handler here to fix the new bug you found.
Previously we check if the cursor is at the top/bottom or left/right side of the middle of the element, and stop checking for self hover if any one of them is true.
After this adjustment, only hovering on the edge (top/bottom or left/right) will trigger the same logic.